os/include/tinyara/fs/mtd.h

```diff
@@ -68,6 +68,9 @@
  * Pre-Processor Definitions
  ****************************************************************************/
 
+#ifdef CONFIG_MTD_PARTITION_NAMES
+#define PART_NAME_MAX           15
```

Comment:
I think you had better keep previous naming. MTD_PARTNAME_LEN

Reply:
Yes. I modified it